index.html

```diff
@@ -1,29 +1,248 @@
+                              </ul>
+                            </div>
+                                  
+                                </div>
+
+         </div>
+         <div class="featuredimg">
+          <div class="container1">
+            <div class="action1">
+            <button class="buton2" type="button"> See Project</button>
+          </div>
+          <ul class="tag1">
+            <li class="ruby1">   Ruby</li>
+            <li class="bootstrap1"> bootstrap</li>
+            <li class="html1"> html</li>
+            <li class="css1"> css</li>
+          </ul>
+         <div class="supporttext11">
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <h2 class="titlpost11">Multi-Post Stories
+            </h2>
+         </div>
+         <div class="imgplace">
+
+        </div>
+         </div>
+        </div>
+    <div class="title-work">
+      <div class="h2r"></div>
+      <h2 class="titl">My Recent Works
+      </h2>  </div> 
+  </section>
+  <section class="aboutdiv">
+    <h2 class="prmtxt">
+      About me
+    </h2>
+    <div class="action2">
+      <button class="buton3" type="button"> Get My Resume</button>
+    </div>
+  <p class="descrpt">
+    Hello I’m a software developer! 
+    I can help you build a product , 
+    feature or website Look through some of my work
+     and experience! If you like what you see and 
+     have a project you need coded, don’t hestiate to contact me.
+
+  </p>
+  
+  <div class="grpimg">
   </div>
-</body>
-</html>
\ No newline at end of file
+  <div class="divider"></div>
+  <div class="skills">
+    <div class="skil2">
+      <ul class="nameskill">
+        <li class="terminal">Terminal</li>
+        <li class="gitlab">Gitlab</li>
+        <li class="codepen">Codepen</li>
+        <li class="github12">Github</li>
+        <li class="codekit">Codedkit</li>
+      </ul>
+      <div class="skit">
+        <div class="rect1"></div>
+        <div class="nametit">Skills</div>
+      </div>
+    </div>
+    <div class="skil1">
+      <ul class="nameskill2">
+        <li class="terminal">Selenium</li>
+        <li class="gitlab">CapyBara</li>
+        <li class="codepen">RSPec</li>
+        <li class="github12">Ruby on Rails</li>
+        <li class="codekit">bootstrap</li>
+      </ul>
+      <div class="skit2">
+        <div class="rect2"></div>
+        <div class="nametit2">Frameworks</div>
+      </div>
+    </div>
+    <div class="skil3">
+        <ul class="nameskill3">
+        <li class="gitlab">Css</li>
+        <li class="codepen">Html</li>
+        <li class="github12">Ruby</li>
+        <li class="codekit">Javascript</li>
+      </ul>
+      <div class="skit3">
+        <div class="rect3"></div>
+        <div class="nametit2">Language</div>
+      </div></div></div></section></body></html>
```

Comment:
Kindly wrap `Languages`, 'Skills' and `Framework` inside a `ul` tag and wrap each one of them inside `li` as the previous reviewer asked, so it will be like:
```
<ul>
   <li>Languages</li>
   <li>Skills</li>
   <li>Framework</li>
<ul>
```

Reply:
I worked on it, and it required a lot of changes, but now I am done.

Reply:
@gemmen29